config/capi/_ytt_lib/capi-k8s-release/values/_default.yml

```diff
@@ -18,6 +18,7 @@ ccdb:
   database: cloud_controller
   host: capi-database-postgresql
   password_secret_name:
+  encryption_key_secret_name: capi-database-encryption-key-secret
```

Comment:
this shouldn't block merging, but why do we configure these in cf-for-k8s when we could hardcode them in capi-k8s-release? I guess it gives some decoupling between the two where we could ostensibly deploy capi-k8s-release on its own, but the coupling is still apparent in the fact we have to make this PR at all...

Reply:
The rationale here is that the secret is defined in cf-for-k8s, so it's the source of truth for the name. We decided this time to give the value in capi-k8s-release a default so it wouldn't break our pipeline while we waited for this PR to merge. We should be able to safely take out the default value after this is merged.

Another option we have is that we could move the Secret definition into capi-k8s-release and avoid the coupling entirely. I'm not sure this meshes with the long term plan for secrets in cf-for-k8s, though.

Reply:
I do now recall this being prescribed by the secrets proposal. With time and distance, I still remember that there are good reasons to have the secrets in cf-for-k8s (they're part of stable user configuration) but I'm not sure why the name needs to be configurable on the capi-k8s-release values interface.

Reply:
@matt-royal Eric and I had the same Q-- why is the name of the secret configurable?